src/compiler/crystal/codegen/codegen.cr

```diff
@@ -5,7 +5,7 @@ require "../semantic/type_inference"
 require "../program"
 require "./llvm_builder_helper"
 
-LLVM.init_x86
+LLVM.init_all_targets
```

Comment:
I'd like to take another shot at using different backends later, however I will remove my backend modification from this PR.